app/models/content_item.rb

```diff
@@ -0,0 +1,53 @@
+class ContentItem
+  attr_reader :content_item, :body, :image, :description, :document_type, :title, :base_path, :locale
+
+  def initialize(content_store_response)
+    @content_item = content_store_response
+    @body = content_item.dig("details", "body")
+    @image = content_item.dig("details", "image")
+    @description = content_item["description"]
+    @document_type = content_item["document_type"]
+    @title = content_item["title"]
+    @base_path = content_item["base_path"]
+    @locale = content_item["locale"]
+  end
+
+  def available_translations
+    translations = content_item["links"]["available_translations"] || []
+
+    mapped_locales(sorted_locales(translations))
+  end
+
+  def sorted_locales(translations)
+    translations.sort_by { |t| t["locale"] == I18n.default_locale.to_s ? "" : t["locale"] }
+  end
+
+  def show_default_breadcrumbs?
+    true
+  end
+
+  def parsed_content_item
+    content_item
+  end
+
+  delegate :to_h, to: :content_item
+  delegate :cache_control, to: :content_item
+
+private
+
+  def mapped_locales(translations)
+    translations.map do |translation|
+      {
+        locale: translation["locale"],
+        base_path: translation["base_path"],
+        text: native_language_name_for(translation["locale"]),
+      }.tap do |h|
+        h[:active] = true if h[:locale] == I18n.locale.to_s
+      end
+    end
+  end
+
+  def native_language_name_for(locale)
```

Comment:
I think this too is a presentation concern.

Reply:
Removed (not necessary for this small change)